change/@fluentui-react-components-ccce747e-48bb-4cfe-a1e6-3b170b7c6f42.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
this should be `patch` as well no ? nothing changed from stable/unstable API perspective

Reply:
Fixed